CONTRIBUTING.md

```diff
@@ -141,6 +141,14 @@ Installing the script `fmt-check` from https://github.com/edsrzf/gofmt-git-hook
 locally as a pre-commit hook checks formatting before committing automatically,
 just copy this script to `.git/hooks/pre-commit`.
 
+The project is using the program
+[`golangci-lint`](https://github.com/golangci/golangci-lint) to run a list of
+linters and checkers. It will be run on the code when you submit a PR. In order
+to check your code beforehand, you can run `golangci-lint run` manually.
+Eventually, we will enable `golangci-lint` for the whole code base. For now,
+you can ignore warnings printed for lines you did not modify, those will be
```

Comment:
It may be better to use the `--new-from-rev upstream/master` flag than to manually ignore errors.

Reply:
Ah, that's a good hint, thanks!

Reply:
Hm, it does not work, I've just tested it locally. I'll leave the hint as it is.

Reply:
Strange. It's work on Windows very well.